Update documentation url and add badges to README

brettlangdon opened the pull request:
Now that `webmock-fixtures` is published to RubyGems we can update the documentation link to point to http://www.rubydoc.info/gems/webmock-fixtures

As well, I added in ruby gem version badge and travis build status badges to the README.

Fixes #5 

/cc @twolfson

twolfson commented:
Now that the documenation is up, I have some questions/feedback:

- The backticks don't seem to be respected here
    - http://www.rubydoc.info/gems/webmock-fixtures/WebMock/Fixtures/Manager#fixtures-class_method
- Can we wrap the `e.g.` to another line somehow? It feels lengthy
    - http://www.rubydoc.info/gems/webmock-fixtures/WebMock/Fixtures/Manager#fixtures-class_method
- This seems to say it returns `Object` when it actually returns nothing (or `nil`). Any idea if we can correct that in Yard?
    - http://www.rubydoc.info/gems/webmock-fixtures/WebMock/Fixtures/Manager#register_fixture-class_method

twolfson commented on README.md:
```diff
@@ -1,10 +1,13 @@
 WebMock fixtures
 ================
+[![Gem Version](https://badge.fury.io/rb/webmock-fixtures.svg)](https://rubygems.org/gems/webmock-fixtures)
+[![Build Status](https://travis-ci.org/underdogio/webmock-fixtures.svg)](https://travis-ci.org/underdogio/webmock-fixtures)
```
We are going to want to use the `master` branch's status. Otherwise, this will show status for the latest build regardless of branch (e.g. `dev/*`, PRs):

``` md
[![Build Status](https://travis-ci.org/underdogio/webmock-fixtures.svg?branch=master)](https://travis-ci.org/underdogio/webmock-fixtures)
```

twolfson commented:
Looks good except for the missing branch fixation

brettlangdon committed:
Update documentation url and add badges to README

brettlangdon commented:
I updated to try and address the comments you had about the documentation.

As well, I noticed that YARD did a pretty bad job of rendering our README. So I added in `.yardopts` to make sure that we use `markdown` formatting and included `LICENSE-MIT` and `CHANGELOG.md` in our documentation as well.

twolfson commented:
For my knowledge, are you able to preview this locally to make sure the updates will work?

twolfson commented:
Changes look good :+1:
